### DISCUSSION TITLE
How to view more lists in a single page?

### DISCUSSION THREAD

**Discussion by Chirunos** [collinbarrett/FilterLists#3372] [Q&A]
So that it's easier to navigate and look for specific lists without using filters

**Comment by collinbarrett** [comment]
Thanks for the question/suggestion. Would you suggest making the rows more compact to fit more? Or, an infinite scroll type design? It's current design does expand to show more rows based on how tall your screen is. But, I'm not a front-end specialist, and I'm sure it can be improved.

**Reply by Chirunos** [reply]
Infinite scroll would be better IMO... or perhaps make an option to view a certain number per page (can be locked to specific numbers like 50, 100 or 200 instead of any set by the user). Sorry for the delay 😅
